tests/test_2.py

```
@@ -0,0 +1,231 @@
+import logging
+from datetime import datetime, timedelta
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+
+# Native AI imports
+from src.domains.agents.observer.ob_agent import ObserverAgent
+from src.domains.agents.analyzer.analyzer_agent import AnalyzerAgent
+from src.domains.agents.decision.decision_agent import DecisionAgent
+from src.domains.tools.calandar_tool import get_calendar_tools, schedule_meeting, find_free_slots
```

Ruff (0.12.2)

`src.domains.tools.calandar_tool.schedule_meeting` imported but unused

Remove unused import

(F401)